PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetEnergyFlow.h

```diff
@@ -56,20 +56,20 @@ class AliAnalysisTaskEmcalJetEnergyFlow: public AliAnalysisTaskEmcalJet {
 	Bool_t			Run()					;
 
 	void			AllocateJetHistograms()			;
-	void			AllocateTrackHistograms()		; // /<Same as Sample task
-	void                    AllocateClusterHistograms()             ; // /<May remove later
-  	void                    AllocateCellHistograms()                ; // /<May remove later
+	void			AllocateTrackHistograms()		; ///<Same as Sample task
+	void                    AllocateClusterHistograms()             ; ///<May remove later
+  	void                    AllocateCellHistograms()                ; ///<May remove later
 	void			AllocateEnergyflowHistograms()		;
 
 	void                    DoJetLoop()                             ;
 	void                    DoTrackLoop()                           ;
 	void			FillEFHistograms()			;
-	void                    DoClusterLoop()                         ; // /<May remove later	
-	void                    DoCellLoop()                            ; // /<May remove later
+	void                    DoClusterLoop()                         ; ///<May remove later	
+	void                    DoCellLoop()                            ; ///<May remove later
 
-        Bool_t                  IsMCprod                                ;// /<Flag for MC productions
-	THistManager            fHistManager                            ;// /!<! Histogram manager
-//	TList*			fOutput					;// /!<! Output list
+        Bool_t                  IsMCprod                                ;///<Flag for MC productions
+	THistManager            fHistManager                            ;///!<! Histogram manager
```

Comment:
This is close, but still won't quite do what you want. You have `///!<!`, but it should be `//!<!` (only two `/`) (Currently, I suspect this will interpret fHistManager as streaming due to the extra `/`, but don't recall precisely). I know I seem perhaps annoyingly specific on this, but streamers can cause a lot of problems, so it can save you a lot of trouble to have them correct from the start

Reply:
Oh thanks, so my situation at the moment is that the train test fails and I was trying to see what causes the problem. I "managed" to replicate the error with my local AliPhysics version so I should probably close this PR until I fix that error. Then you won't be spammed by notifcations right? Thank you for your help so far Raymond! :)

Reply:
Sure, that sounds like a good plan. I'll look for your PR when it's ready. You're welcome!